pennylane/templates/subroutines/qmc.py

```diff
@@ -74,74 +74,58 @@ def probs_to_unitary(probs):
     return unitary
 
 
-def func_to_unitary(xs, func):
+def func_to_unitary(func, M):
     r"""Calculates the unitary that encodes a function onto an ancilla qubit register.
 
-    Consider the set of :math:`M` points :math:`X = \{x_{0}, x_{1}, \ldots, x_{M - 1}\}` and the
-    corresponding :math:`d`-dimensional discrete space :math:`X^{d}`. Also, consider the function
-    :math:`f: X^{d} \rightarrow \mathbb{R}` such that :math:`0 \leq f(x) \leq 1` for all
-    :math:`x \in X`.
+    Consider a function defined on a set of integers :math:`X = [0, 1, \ldots, M - 1]` whose output
+    is bounded in the interval :math:`[0, 1]`, i.e., :math:`f: X \rightarrow [0, 1]`.
```

Comment:
The set of integers are not necessarily the values that the user wants to use in the function, correct?

For example, I may be using a normal probability distribution with mean 10 and truncating it to [-5,5]. Then these integers X = [0, 1, ... M-1] will correspond to [-5,...5] and the user has to make sure to account for this in the function. E.g. 
```
def func(x):
return sin(x)
```
Must become something like:
```
def func(x):
return sin((10*x/(M-1)-5))
```
I'm wondering how we can make this clear and where to include it.

Reply:
Good point! Yes that is true, hopefully the example in the usage details can make it clear one approach to dealing with this.